invenio_app_ils/documents/mappings/os-v1/documents/document-v1.0.0.json

```diff
@@ -2,24 +2,48 @@
   "settings": {
     "analysis": {
       "normalizer": {
-        "lowercase_normalizer": {
+        "custom_normalizer": {
           "type": "custom",
-          "char_filter": [],
+          "char_filter": [
+            "strip_special_chars"
+          ],
           "filter": [
-            "lowercase"
+            "lowercase",
+            "asciifolding"
           ]
         }
       },
       "analyzer": {
         "min_length_2": {
           "tokenizer": "standard",
           "filter": [ "min_length_2" ]
+        },
+        "custom_analyzer": {
+          "tokenizer": "standard",
+          "type": "custom",
+          "char_filter": [
+            "strip_special_chars"
+          ],
+          "filter": [
+            "lowercase",
+            "asciifolding"
+          ]
         }
       },
       "filter": {
         "min_length_2": {
           "type": "length",
           "min": 2
+        },
+        "asciifolding": {
+          "type": "asciifolding"
+        }
+      },
+      "char_filter": {
+        "strip_special_chars": {
+          "type": "pattern_replace",
+          "pattern": "[\\p{Punct}\\p{S}]",
```

Comment:
Update this with an exhaustive list of special chars if that is preferred.

Reply:
maybe a regex already findable in some documentation?

Reply:
I used this for reference: https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html